docs/index.rst

```diff
@@ -42,19 +40,20 @@ This website contains the draft specification of the Beneficial Ownership Data S
 
    .. cssclass:: primaryfrontpageboxlink
 
-      :doc:`Visit the Data Schema <../schema/index>`
+      :doc:`Key Concepts <../schema/concepts>`
 
    .. cssclass:: secondaryfrontpageboxlink
 
-      :doc:`The data model  <../schema/concepts>`
+      :doc:`Modelling requirements  <../schema/modelling-requirements>`
```

Comment:
When I build this locally, the link for 'Modelling requirements' and 'system requirements' doesn't show.

Reply:
oops yes I should have mentioned that - these sections don't exist yet. Once the docs are restructured those should work.

Reply:
@kathryn-ods - I think my preference is to just remove the links entirely for now so that we don't have the build warnings on main. (When we restructure the docs, we can also set up anchors and use :ref: s here instead of :doc:s so that we don't need to put paths in links.)

Reply:
will do!